components/bl-collapsiblePanel-component/README.md

```diff
@@ -0,0 +1,83 @@
+# CollapsiblePanel
+
+CollapsiblePanel is the component that can be used in Backendless [UI-Builder](https://backendless.com/developers/#ui-builder). This component displays summary information and upon selecting the panel, it expands to display the full expansion panel.
+
+<p align="center">
+  <img src="./thumbnail.png" alt="main thumbnail" width="780"/>
+</p>
+
+## Properties
+
+| Property                           | Type    | Default value | Logic                         | Data Binding | UI Setting | Description                                           |
+|------------------------------------|---------|---------------|-------------------------------|--------------|------------|-------------------------------------------------------|
+| Width <br> `width`                 | *Text*  |               |                               | NO           | YES        | Controls the width of the panel.                      |
+| Title <br> `titleDefault`          | *Text*  | `Panel Title` | Title Logic                   | YES          | YES        | The logic to determine the title of the panel.        |
+| Title On Open <br> `titleOpen`     | *Text*  |               | On Open The Panel Title Logic | YES          | YES        | The logic to determine the title on opened the panel. |
+| Title Background <br> `background` | *Color* |               |                               | NO           | YES        | Controls the background color of the panel title.     |
+| Title Color <br> `color`           | *Color* |               |                               | NO           | YES        | Controls the color of the panel title.                |
+| Title Font Size <br> `fontSize`    | *Text*  |               |                               | NO           | YES        | Controls the font size of the panel title.            |
+| Title Padding <br> `padding`       | *Text*  |               |                               | NO           | YES        | Controls the padding of the panel title.              |
+| Icon Color <br> `iconColor`        | *Color* |               |                               | NO           | YES        | Controls the color of the panel title icon.           |
+
+## Events
+
+| Name                 | Triggers                                          | Context Blocks |
+|----------------------|---------------------------------------------------|----------------|
+| On Open Panel Event  | Triggered when the user opens the panel section.  |                |
+| On Close Panel Event | Triggered when the user closes the panel section. |                |
+
+## Actions
+
+| Action       | Inputs | Returns |
+|--------------|--------|---------|
+| Show Content |        |         |
+| Hide Content |        |         |
```

Comment:
The UI builder appends the component name to the end of action label, so we take that into account in readme/component.json, and use `of/from/to...` if needed.